src/parkapi_sources/converters/apcoa/converter.py

```diff
@@ -0,0 +1,70 @@
+"""
+Copyright 2024 binary butterfly GmbH
+Use of this source code is governed by an MIT-style license that can be found in the LICENSE.txt.
+"""
+
+import requests
+from validataclass.exceptions import ValidationError
+from validataclass.validators import DataclassValidator
+
+from parkapi_sources.converters.base_converter.pull import PullConverter
+from parkapi_sources.exceptions import ImportParkingSiteException
+from parkapi_sources.models import RealtimeParkingSiteInput, SourceInfo, StaticParkingSiteInput
+
+from .mapper import ApcoaMapper
+from .validators import ApcoaParkingSiteInput
+
+
+class ApcoaPullConverter(PullConverter):
+    _base_url = 'https://api.apcoa-services.com/carpark-dev/v4'
+    required_config_keys = ['PARK_API_APCOA_API_SUBSCRIPTION_KEY']
+
+    mapper = ApcoaMapper()
+    apcoa_parking_site_validator = DataclassValidator(ApcoaParkingSiteInput)
+
+    source_info = SourceInfo(
+        uid='apcoa',
+        name='APCOA-SERVICES API',
+        public_url='https://devzone.apcoa-services.com/',
+        has_realtime_data=False,  # ATM only static data can be called from the API
+    )
+
+    def get_static_parking_sites(self) -> tuple[list[StaticParkingSiteInput], list[ImportParkingSiteException]]:
+        static_parking_site_inputs: list = []
+        static_parking_site_errors: list[ImportParkingSiteException] = []
+
+        parking_site_dicts = self.get_data()
+
+        for parking_site_dict in parking_site_dicts.get('Results', []):
```

Comment:
Missing validation: what happens, if there is no dict? -> ListValidator with AnythingValidator enforcing dicts inside the list.